dev-manual/api/api-reference-archivematica.rst

```diff
@@ -484,9 +484,9 @@ Response definitions:
 
 ==================   ==========================================================
 
-``message``          "Approval successful."
+``error``            "false"
 
-``reingest_uuid``    UUID of the reingested transfer.
+``message``          "Metadata files added successfully."
```

Comment:
```suggestion
``message``          Success message or reason for the error.
```

Reply:
Thanks for the suggestion. Let me work on it.